Add missing conversation programs to campus dictionary

zimmer18 opened the pull request:
Added 
- Asian Conversations
- Environmental Conversation
- Public Affairs Conversation
- Science Conversation

to the campus dictionary and changed abbreviation of Am Con to match all other abbreviations.

Commit:
Add missing conversation programs to campus dictionary

hawkrives commented on data/dictionary/asian-conversations.yaml:
```diff
@@ -0,0 +1,11 @@
+word: Asian Conversations
```
Shouldn't this be Asian Con, to match the others?

zimmer18 replied:
Yup!

hawkrives commented on data/dictionary/amcon.yaml:
```diff
@@ -1,4 +1,4 @@
-word: Amcon
+word: Am Con
```
Capitalize the Con part to match the others?

hawkrives replied:
Was this already capitalized and I misread it? Looks like you can ignore this comment.

hawkrives replied:
Please change this to "AmCon" as we discussed separately.

hawkrives reviewed (changes requested):
Two tweaks maybe?

Commit:
Fix Am Con and Asian Con abbreviations

Amcon was used throughout the text and those are all changed to Am Con now
Asian Conversation was shortened to match the other conversations

hawkrives commented on data/dictionary/amcon.yaml:
```diff
@@ -1,11 +1,11 @@
   program and learning community in which students analyze America’s history
   and culture. Over four successive semesters in their first and sophomore
-  years, Amcon students encounter many of the texts, topics, and historical
+  years, Am Con students encounter many of the texts, topics, and historical
```
... and fix it in here too, plz?

hawkrives commented on data/dictionary/asian-conversations.yaml:
```diff
@@ -0,0 +1,11 @@
+word: Asian Con
```
s/Asian Con/AsianCon/

hawkrives commented on data/dictionary/encon.yaml:
```diff
@@ -0,0 +1,9 @@
+word: En Con
```
Um. EnCon, unless they followed in the footsteps of Great Con?

You might need to find someone and ask them.

hawkrives commented on data/dictionary/paccon.yaml:
```diff
@@ -0,0 +1,18 @@
+word: Pac Con
```
Find someone on-campus and ask? PaCon/PacCon/Pac Con/I dunno

hawkrives commented on data/dictionary/scicon.yaml:
```diff
@@ -0,0 +1,15 @@
+word: Sci Con
```
SciCon

hawkrives reviewed (changes requested):
As @zimmer18 and I discussed in Slack, we're going to switch these to the on-campus colloquial versions.

Commit:
Change all conversation programs to their typical campus spellings

Commit:
Fix dictionary.json file to match with updated yaml files

hawkrives reviewed (approved):
Thanks!
